dipy/viz/streamline.py

```diff
@@ -1,12 +1,18 @@
+import warnings
 from dipy.utils.optpkg import optional_package
 
 plt, have_plt, _ = optional_package("matplotlib.pyplot")
 fury, has_fury, _ = optional_package('fury', min_version="0.10.0")
 if has_fury:
     from dipy.viz import actor, window
 
+sagital_deprecation_warning_msg = (
```

Comment:
can you update the variable name with the correct syntax:
`sagittal_deprecation_warning_msg` instead of `sagital_deprecation_warning_msg`

Reply:
Changed in b5be3fc.